apps/meteor/app/statistics/server/lib/statistics.ts

```diff
@@ -546,27 +546,27 @@ export const statistics = {
 		statistics.messageAuditLoad = settings.get('Message_Auditing_Panel_Load_Count');
 		statistics.joinJitsiButton = settings.get('Jitsi_Click_To_Join_Count');
 		statistics.slashCommandsJitsi = settings.get('Jitsi_Start_SlashCommands_Count');
-		statistics.totalOTRRooms = await Rooms.findByCreatedOTR().count();
+		statistics.totalOTRRooms = await Rooms.countByCreatedOTR();
```

Comment:
it would be really good if we can add the `readPreference` for every other queries as well